src/System.IO.FileSystem.AccessControl/src/System/Security/AccessControl/DirectorySecurity.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+
+namespace System.Security.AccessControl
+{
+    public sealed class DirectorySecurity : FileSystemSecurity
+    {
+        public DirectorySecurity()
+            : base(true)
+        {
+        }
+
+        public DirectorySecurity(string name, AccessControlSections includeSections)
+            : base(true, name, includeSections, true)
+        {
+            Path.GetFullPath(name);
```

Comment:
This call is useless, you can remove it.

Reply:
It isn't. It will throw if there is an embedded null. We _should_ make this more explicit, but I didn't want to include any functional changes in this PR.